### ISSUE TITLE
Include adders from `svelte-add` (v2)

### ISSUE THREAD

**Issue by manuel3108** [svelte-society/sveltesociety.dev#607]
A few days ago, @benmccann came up with the idea of integrating `svelte-add` into the new v2 site. The goal would be to make people aware of that this project exists and potentially directly link them from the official svelte / kit sites.

Based on my recent commitment to a full rewrite of svelte-add, I personally really like that idea. The rewrite is currently usable, although published as `svelte-compose` as im still in the process of getting access to the `svelte-add` npm org.

I could imagine having something like on the website I have currently published https://svelte-compose.com/. 
![image](https://github.com/svelte-society/sveltesociety.dev/assets/30698007/ec476a43-e661-4406-8af0-570a1f80afb5)

We would need to decide if the configurator displayed below would make sense in this context, but I think especially for newcomers this takes away a lot of uncertainty.
![image](https://github.com/svelte-society/sveltesociety.dev/assets/30698007/c2c82bf4-5dc2-47a4-9343-50013c48be0b)

What do you think? Would this be an interesting addition? Would this addition work well with the packages & templates shown on the current website?

---

Technical details:
Currently one of the packages (`@svelte-compose/cli`) is exposing an internal API to get a list of adders and their respective metadata. Based on that information the page is fully prerendered during build and then published to github pages. If the separate website would need to co-exists remains unclear at this point.

**Comment by kevmodrome** [comment]
Sorry for the late reply here. I believe this could live as a collection on the new site. We could make it prominent by adding it to the sidebar on the left. Do you think that's enough should we make this a separate page altogether?

**Comment by manuel3108** [comment]
If I understood correctly, the adders will also be featured on the new docs site, so I think the importance of this has dramatically decreased. I think having it on there might still be interesting, but we should probably avoid duplicating content from the upcoming new docs site.

**Comment by benmccann** [comment]
We decided in the past couple of months to make Svelte Add an official project, so the docs will end up on svelte.dev. The other thing that makes listing them on sveltesociety.dev tricky is that they all live in a single repo, which doesn't really fit the current setup where there's generally a repo per package. I removed a lot of the old adders awhile back in https://github.com/svelte-society/sveltesociety.dev/commit/c549e9cd5708aa9045e994c451507eea5045b9ce.
